### ISSUE TITLE
Fix https mixed content issue (cont.) 

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/biocollect#519]
This follows from work on #459 

When Matt Andrews enables https for ala services biocollect depends on, we need to update the configuration settings below to use the https version of such services

biocache.baseURL=http://biocache.ala.org.au
spatial.baseURL=http://spatial.ala.org.au
sld.polgon.default.url=http://fish.ala.org.au/data/alt-dist.sld
sld.polgon.highlight.url=http://fish.ala.org.au/data/fc-highlight.sld

lists.baseURL=http://lists.ala.org.au

Additionally web-app/js/dashboard.js also needs to be changed to use the https version for spatial.org.au

**Comment by matthewandrews** [comment]
https://spatial.ala.org.au is now available.

**Comment by matthewandrews** [comment]
https://biocache.ala.org.au is also now available.

**Comment by javier-molina** [comment]
Thanks Matt

I will update configs for biocollect to include those services shortly

**Comment by matthewandrews** [comment]
We also now have https://lists.ala.org.au/

**Comment by matthewandrews** [comment]
And now https://fish.ala.org.au (but the CSS and some theme images are hardcoded to http so don't load in most browsers over https).

**Comment by javier-molina** [comment]
Ey Matt do you think you can add those to your plate?

I spotted a few resources (that were beyond the actual application) that biocollect sources from http

When I try
https://biocollect-test.ala.org.au/images/filetypes/twitter.png 

the browser will redirect me to 

http://biocollect-test.ala.org.au/static/images/filetypes/twitter.png

I am guessing that is an apache redirect that is ignoring the original protocol

Similar for production, the url below
https://biocollect.ala.org.au/images/filetypes/twitter.png 

is redirected to 
http://biocollect.ala.org.au/static/4AeKmcXRBFRkGwnz80KXEIyhlmXsrsNgA35zy5YT6jM.png 

Thanks

**Comment by matthewandrews** [comment]
I don't see an Apache redirect for this... Apache is pretty much just proxying everything to Tomcat on 8080. Looks like the response is from the app - perhaps because the internal proxying is via http it just reuses that - perhaps the app should force the static resource responses to https.

**Comment by javier-molina** [comment]
We will need to investigate in the tomcat side then. I spend the whole day
yesterday and I couldn't find anything in the app side. I don't see that
redirect happening in my local environment so I didn't see this issue
before.

**Comment by javier-molina** [comment]
@matthewandrews  after further investigation, looks like the grails resources plugin creates that http redirect (instead of https) because it does not see the original call information given that apache is proxying the requests.

One suggestion is to use  the Remote Valve as per comments in this  [thread](https://groups.google.com/a/cloudfoundry.org/forum/#!topic/vcap-dev/4OnLZlP7NvU). I tried that but I must be missing something. 

In any case Sathish mentioned that the idea is to do the same as in merit, ie, enforce/redirect to https if someone tries to access the service via http, that would solve the problem with the static resources redirect.

Long story short, can you please force http to https redirection for biocollect test? Once done I will test and confirm that everything works as it should.

Thanks

**Comment by javier-molina** [comment]
@matthewandrews  another couple of services, not available over https, for when you have a chance

http://collections.ala.org.au/ 
http://images.ala.org.au/

**Comment by matthewandrews** [comment]
https://images.ala.org.au/ should be available.
However collections will be tricky, as it's running through an IM&T proxy at the moment.
